components/Icons/CoinIcon/index.js

```diff
@@ -10,26 +11,21 @@ const useStyles = makeStyles(() => ({
     display: 'flex'
   },
   img: props => ({
-    height: props.height,
+    height: props.size,
```

Comment:
`size` usually represents  2 values: `width, height` but it seems that you are using it for presenting `height` only. I believe the use of `size` here is miss-leading, unless am missing some internal functionality.

Reply:
![image](https://user-images.githubusercontent.com/78897434/134091883-fab6a61c-d0fd-4467-bae4-b4214abab0b5.png)
The "size" is used as width and height props values for Image component.
@saito-sv